projects/scion/microfrontend-platform/src/lib/host/manifest-registry/ɵmanifest-registry.ts

```diff
@@ -60,7 +60,7 @@ export class ɵManifestRegistry implements ManifestRegistry, PreDestroy {
   public resolveCapabilitiesByIntent(intent: Intent, appSymbolicName: string): Capability[] {
     assertExactQualifier(intent.qualifier);
     const filter: ManifestObjectFilter = {type: intent.type, qualifier: intent.qualifier || {}};
-    return this._capabilityStore.find(filter, capabilityQualifier => new QualifierMatcher(capabilityQualifier, {evalAsterisk: true, evalOptional: true}).matches(intent.qualifier))
+    return this._capabilityStore.find(filter, capabilityQualifier => new QualifierMatcher(capabilityQualifier, {evalAsterisk: false}).matches(intent.qualifier))
```

Comment:
Why are we passing the qualifier in the filter? I am asking because below, we are passing a qualifier predicate.
Or better, I think, here, we do not need to pass a qualifier predicate at all, simply pass the qualifier in the filter.